src/async/Engine/Engine.js

```diff
@@ -73,9 +73,8 @@ module.exports = class Engine {
           ? this.client.startWorkflow(job)
           : this.client.startTask(job);
 
-        await handler;
-
-        return undefined;
+        const res = await handler;
+        return res;
```

Comment:
You can just return handler: `return handler`

Reply:
@jalric 
That what I thought but I have the following warning if I do so:
https://eslint.org/docs/rules/no-return-await

Reply:
no need to return await, just return the handler.